app/views/surveys/_statement.html.erb

```diff
@@ -0,0 +1,8 @@
+<div id = "statement_<%= params["question_count"] %>" class="form-group">
```

Comment:
this partial render question then why file name is statement?

Reply:
In db the column name for question is statement